plugins/inputs/vsphere/README.MD

```diff
@@ -0,0 +1,350 @@
+in order to avoid potential errors for time elapsed during a collection cycle. This should never be greater than 8,
+though the default of 1 (no concurrency) should be sufficient for most configurations. 
+
+## Measurements &amp; Fields
+
+- Cluster Stats
+	- Cluster services: CPU, memory, failover
+	- CPU: total, usage
+	- Memory: consumed, total, vmmemctl
+	- VM operations: # changes, clone, create, deploy, destroy, power, reboot, reconfigure, register, reset, shutdown, standby, vmotion
+- Host Stats:
+	- CPU: total, usage, cost, mhz
+	- Datastore: iops, latency, read/write bytes, # reads/writes
+	- Disk: commands, latency, kernel reads/writes, # reads/writes, queues
+	- Memory: total, usage, active, latency, swap, shared, vmmemctl
+	- Network: broadcast, bytes, dropped, errors, multicast, packets, usage
+	- Power: energy, usage, capacity
+	- Res CPU: active, max, running
+	- Storage Adapter: commands, latency, # reads/writes
+	- Storage Path: commands, latency, # reads/writes
+	- System Resources: cpu active, cpu max, cpu running, cpu usage, mem allocated, mem consumed, mem shared, swap
+	- System: uptime
+	- Flash Module: active VMDKs 
+- VM Stats:
+	- CPU: demand, usage, readiness, cost, mhz
+	- Datastore: latency, # reads/writes
+	- Disk: commands, latency, # reads/writes, provisioned, usage
+	- Memory: granted, usage, active, swap, vmmemctl
+	- Network: broadcast, bytes, dropped, multicast, packets, usage
+	- Power: energy, usage
+	- Res CPU: active, max, running
+	- System: operating system uptime, uptime
+	- Virtual Disk: seeks, # reads/writes, latency, load 
+- Datastore stats:
+	- Disk: Capacity, provisioned, used  
+
+For a detailed list of commonly available metrics, please refer to [METRICS.MD](METRICS.MD)
+	
+## Tags
+
+- all metrics
+	- vcenter (vcenter url)
+- all host metrics
+	- cluster (vcenter cluster)
+- all vm metrics
+	- cluster (vcenter cluster)
+	- esxhost (name of ESXi host)
+	- guest (guest operating system id)
+- cpu stats for Host and VM
+	- cpu (cpu core - not all CPU fields will have this tag)
+- datastore stats for Host and VM
+	- datastore (id of datastore)
+- disk stats for Host and VM
+	- disk (name of disk)
+- disk.used.capacity for Datastore
+	- disk (type of disk)
+- net stats for Host and VM
+	- interface (name of network interface)
+- storageAdapter stats for Host
+	- adapter (name of storage adapter)
+- storagePath stats for Host 
+	- path (id of storage path)
+- sys.resource* stats for Host
+	- resource (resource type)
+- vflashModule stats for Host
+	- module (name of flash module)
+- virtualDisk stats for VM
+	- disk (name of virtual disk)
+
+## Sample output
+
+```
+vsphere_vm_cpu,esxhostname=DC0_H0,guest=other,host=host.example.com,moid=vm-35,os=Mac,source=DC0_H0_VM0,vcenter=localhost:8989,vmname=DC0_H0_VM0 run_summation=2608i,ready_summation=129i,usage_average=5.01,used_summation=2134i,demand_average=326i 1535660299000000000
```

Comment:
I think this is a couple collections, lets remove it down to just one interval.

Reply:
OK. Will check on that.